ascii-dev/mt-react
# #159718068 Setup HoundCI

#### What does this PR do?
To configure HoundCI for the application.
#### Description of Task to be completed?
- create houndci config file
#### How should this be manually tested?
After cloning the repo and running `npm install`
You should see the `.hound.yml` file in the project
#### What are the relevant pivotal tracker stories?
[#159718068](https://www.pivotaltracker.com/story/show/159718068)
#### Questions
N/A
